packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -93,15 +104,15 @@ export const IndexesToolbar: React.FunctionComponent<IndexesToolbarProps> = ({
                       props
                     )}
                   >
-                    <Button
-                      data-testid="open-create-index-modal-button"
-                      disabled={!isWritable}
-                      onClick={onClickCreateIndex}
-                      variant="primary"
-                      size="small"
-                    >
-                      Create Index
-                    </Button>
+                    <CreateIndexButton
+                      isSearchManagementActive={isSearchManagementActive}
+                      isAtlasSearchSupported={isAtlasSearchSupported}
```

Comment:
(nit) Given how we use them we can merge these props into one, right? `isSearchSupported` or something?

Reply:
I was thinking to have both of them separated because one is the feature toggle, and the other one is if Atlas Search is available, so for me it was easier to understand it. I suppose at the end (in a few weeks hopefully haha) isSearchManagementActive will be removed.